test/BitcoinSPVTest.js

```diff
@@ -1,25 +1,19 @@
-/*
-const BitcoinSPV = require("../src/BitcoinSPV").BitcoinSPV
-const ElectrumClient = require("../src/ElectrumClient")
-const config = require("../../../src/config/config.json")
-
-const fs = require("fs")
-const chai = require("chai")
-const assert = chai.assert
+import { BitcoinSPV } from "../src/lib/BitcoinSPV.js"
+import ElectrumClient from "../src/lib/ElectrumClient.js"
+import fs from "fs"
+import chai from "chai"
```

Comment:
We generally import these at the global level rather than invoking them on `chai`. Is the reason here to satisfy TypeScript? I seem to recall having run into an issue or two with TypeScript, chai, and tests, but remember figuring out how to deal with it… Knowledge that is lost but likely easily found hehe.

Reply:
To be honest I don't really remember the exact reason why I went with this. I'll look into doing it in other ways :ok_hand: